.craft.yml

```diff
@@ -0,0 +1,16 @@
+github:
+  owner: getsentry
+  repo: chartcuterie
+
+statusProvider:
+  name: github
+artifactProvider:
+  name: github
+
+targets:
+  - id: release
```

Comment:
I'd either provide an `id` for all targets or none

Reply:
👍